### DISCUSSION TITLE
How can I get the billing address of an order with SP API

### DISCUSSION THREAD

**Discussion by julienBcv** [amzn/selling-partner-api-models#3526] [General]
Hello,
Can someone tell me if it is possible to get the billing address of an order ? with SP API ?

I can get the shipping address of my orders via the SP-API getOrderAddress but I Can't get the billing address.

And I can't find any information about it on the web. Can we do it ? Do I need to set up a specific role? Should I get a specific API?

Thank you for your answers.

**Comment by Kepler-Elliott** [comment]
你好我想问一下这个收货地址怎么拿全啊，我拿到的数据会有些参数没有数据，比如收货的街道1姓名等等

**Comment by Kepler-Elliott** [comment]
Hello, I want to ask how to get this delivery address all ah, I get the data will have some parameters without data, such as receiving street 1 name and so on

**Comment by ritikacd** [comment]
Hi @julienBcv

You can retrieve [Buyer’s information](https://developer-docs.amazon.com/sp-api/docs/orders-api-v0-reference#orderbuyerinfo) through Orders APIs and reports but billing address is not provided. In the TR marketplace, you can get [BuyerTaxInformation](https://developer-docs.amazon.com/sp-api/docs/orders-api-v0-reference#buyertaxinformation) for business merchant which includes BuyerBusinessAddress. 

Please note that the buyer’s data is a restricted data and you will need to use Restricted Data Token (RDT) and restricted developer roles to access this restricted data.

Thanks,
Ritika
Selling Partner API Services
